tests/output/staging/60008-UNI-17253.cs

```diff
@@ -1,4 +1,4 @@
-// It adds an extra newline after each case 
+// It adds an extra newline after each case
```

Comment:
This change should be reverted. Creates noise.

Reply:
if we don't remove space in the comment , this test case will always fail.
so changed the output file.

NOTE :: uncrustify will remove the extra spaces in the end of a comment line